### ISSUE TITLE
Xamarin.iOS issue with azure refresh token

### ISSUE THREAD

**Issue by PeterKolstad** [Azure/azure-mobile-apps-net-client#293]
I login to google with the following command:  
user = await CurrentClient.LoginAsync(viewCtrl, MobileServiceAuthenticationProvider.Google, new Dictionary<string, string>() { { "access_type", "offline" } });
which works perfectly, but when i refresh the token with the following command: 
 user = await CurrentClient.RefreshUserAsync();
I get the following error: 
Refresh failed with a 403 Forbidden error. The refresh token was revoked or expired

The RefreshUserAsync method works perfect with xamarin.andriod but not with Xamarin.iOS. 
What can be wrong?

**Comment by adrianhall** [comment]
Can you turn on diagnostic logging and post any error messages that are written on the backend through the log stream?

**Comment by PeterKolstad** [comment]
Hi, thanks for your quick reply. I finally managed to solve my issue. It appeared to be an issue with my google account that i used for logging in. (I used different accounts when i tested on Android and on iOS). I had to go to https://www.google.com/settings/u/1/security and under "Connected apps" i had three versions of my app installed (different application names). I removed them all and after that everything worked as a charm :) 
Thanks for making me realise that it wasn't anything on the back end and that it had to be something else. I have been struggling with this for about a week so your door opener did the trick :)
